packages/docs-app/src/common/films.tsx

```diff
@@ -178,6 +178,27 @@ export const renderCreateFilmOption = (
     />
 );
 
+export const renderCreateFilmOptions = (
```

Comment:
minor: I think the pluralization in this name is slightly off, it should be:

```suggestion
export const renderCreateFilmsOption = (
```